src/mesh/mesh.jl

```diff
@@ -194,9 +213,7 @@ function generate_mesh()
   end
 
   # Partition mesh
-  if mpi_isparallel()
-    partition!(mesh)
-  end
+  partition!(mesh)
```

Comment:
To be honest, I'd prefer the version with an explicit  `if` condition. This indicates to a new users who is aware of MPI but does not do parallel runs that this is not a method to be concerned with. Otherwise you'd have to look up the method definitions for `partition!` and figure out what is done here, even if only used in serial.

If you feel strongly about this, I would prefer to at least add a comment such as
```julia
# Partition the mesh among multiple MPI ranks (does nothing if run in serial)
```

Reply:
I've added your comment in 0982d66. I don't feel too strongly about this -  we can talk later and discuss questions like this one.